src/DebugBridges/AbstractDebugBridge.ts

```diff
@@ -109,6 +109,13 @@ export abstract class AbstractDebugBridge implements DebugBridge {
         this.updateRuntimeState(this.history[this.present]);
     }
 
+    public reverseContinue() {
+        // Time travel backward
+        // TODO: This should actually implement going back to the previous breakpoint
+        this.present = 0;
+        this.updateRuntimeState(this.history[this.present]);
```

Comment:
TODO: add a method to do this  `updatePresent(newPresent)` that changes the value of `present` and calls the updateRuntimeState